app/src/components/map/FeaturePopup.tsx

```diff
@@ -0,0 +1,241 @@
+  };
+
+  const handlePrev = () => {
+    if (isEmpty) {
+      return;
+    }
+    if (currentIndex === 0) {
+      setCurrentIndex(data.length - 1);
+    } else {
+      setCurrentIndex(currentIndex - 1);
+    }
+  };
+
+  const ModalContentWrapper: React.FC<any> = ({ children }) => <div className={classes.modalContent}>{children}</div>;
+
+  const MetadataHeader: React.FC<React.PropsWithChildren<IMetadataHeaderProps>> = (headerProps) => {
+    const { date, index, length } = headerProps;
+
+    return (
+      <Box pt={0.5} pb={0.75}>
+        <Typography variant="h5" component="h3" className={classes.pointType}>
+          {length && length > 1 ? `Records (${(index || 0) + 1} of ${length})` : 'Record'}
+        </Typography>
+        {date && (
+          <Typography className={classes.date} variant="subtitle1">
+            {getFormattedDate(DATE_FORMAT.ShortMediumDateFormat, date)}
+          </Typography>
+        )}
+      </Box>
+    );
+  };
+
+  const NoMetadataAvailable: React.FC = () => (
+    <Typography className={classes.date} variant="body1">
+      No metadata available.
+    </Typography>
+  );
+
+  if (metadataLoader.isLoading) {
+    return (
+      <ModalContentWrapper>
+        <div className={classes.loading}>
+          <CircularProgress size={24} color="primary" />
+        </div>
+      </ModalContentWrapper>
+    );
+  }
+
+  if (data.length === 0) {
+    return (
+      <ModalContentWrapper>
+        <NoMetadataAvailable />
+      </ModalContentWrapper>
+    );
+  }
+
+  const metadata = data[currentIndex];
+  const type = metadata?.type;
+  const dwc: Record<string, unknown> = metadata?.dwc || {};
+  const filteredMetadata = Object.entries(COMMON_METADATA_PROPERTIES).filter(([key]) => Boolean(dwc[key]));
+
+  if (!dwc || !Object.keys(dwc).length) {
+    return (
+      <ModalContentWrapper>
+        <MetadataHeader type={type} />
+        <NoMetadataAvailable />
+      </ModalContentWrapper>
+    );
+  }
+
+  return (
+    <ModalContentWrapper>
+      <MetadataHeader type={type} index={currentIndex} length={data.length} />
+      <Divider></Divider>
+      <Collapse in={metadataLoader.isReady}>
+        <Table>
+          <TableBody>
+            {filteredMetadata.map(([key, propertyName]) => {
+              return (
+                <TableRow key={key}>
+                  <TableCell>{propertyName}</TableCell>
+                  <TableCell>{(formatMetadataProperty[key] || String)(dwc[key])}</TableCell>
+                </TableRow>
+              );
+            })}
+          </TableBody>
+        </Table>
+      </Collapse>
+      <Box display="flex" justifyContent="space-between" mt={1.5}>
+        {!isEmpty && data.length > 1 && (
+          <Box display="flex">
+            <Button
+              size="small"
+              variant="text"
+              startIcon={<Icon path={mdiChevronLeft} size={1} />}
+              onClick={() => handlePrev()}>
+              PREV
```

Comment:
I think (for the sake of accessability) the contents of these two buttons should be written in Pascal case (`Prev` instead of `PREV`), with the added style `style={{ textTransform: 'uppercase' }}`